server/index.ts

```diff
@@ -0,0 +1,10 @@
+import createApp from './app'
+import HmppsAuthClient from './data/hmppsAuthClient'
+import UserService from './services/userService'
+
+const hmppsAuthClient = new HmppsAuthClient()
+const userService = new UserService(hmppsAuthClient)
+
+const app = createApp(userService)
```

Comment:
In use of force we separated out the wiring up of services from creating the app and that has quite a few benefits - you can create a `Services` type which you can then use as a simple IOC container - which you pass through to the route registering components